react-firebase-chat-master/src/firebase-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "apiKey": "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
+  "authDomain": "chat-app-15946.firebaseapp.com",
+  "projectId": "chat-app-15946",
+  "storageBucket": "chat-app-15946.appspot.com",
+  "messagingSenderId": "533178884935",
+  "appId": "1:533178884935:web:ef9483d776d0107ffe6262",
+  "measurementId": "G-P58VBCQ0SY"
+}
```

Comment:
Incorrect Firebase Token

Reply:
it was moved to a different file so you could have defined it separately in your local machine.